From 7f3f12651fed64522091366fa4c174ea14958dc2 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Quang=20Ng=C3=B4?= <ngoquang2708@gmail.com>
Date: Sat, 1 Oct 2016 18:42:30 +0700
Subject: [PATCH] Allow board to provide libreference-ril

This also fix breaking ninja if BOARD_PROVIDES_LIBRIL is set
while libreference-ril still refers to libril, which is not exists.

Set both BOARD_PROVIDES_LIBRIL and BOARD_PROVIDES_LIBREFERENCE_RIL
to true make ninja happy.

Change-Id: I4f164178ae1c77dae420544ba7cadef8cb7ea883
---
 reference-ril/Android.mk | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/reference-ril/Android.mk b/reference-ril/Android.mk
index d9908c1..551ae65 100644
--- a/reference-ril/Android.mk
+++ b/reference-ril/Android.mk
@@ -1,5 +1,7 @@
 # Copyright 2006 The Android Open Source Project
 
+ifneq ($(BOARD_PROVIDES_LIBREFERENCE_RIL),true)
+
 # XXX using libutils for simulator build only...
 #
 LOCAL_PATH:= $(call my-dir)
@@ -45,3 +47,5 @@ else
   LOCAL_MODULE:= reference-ril
   include $(BUILD_EXECUTABLE)
 endif
+
+endif # BOARD_PROVIDES_LIBREFERENCE_RIL
-- 
1.9.1

